zapier/introduction/zapier-filtering/zapier-adding-filters.md

```diff
@@ -0,0 +1,46 @@
+---
+author: kapnobatai136
+
+category: must-know
+
+aspects:
+  - introduction
+
+type: normal
+
+---
+
+# Adding Filters
+
+---
+## Content
+
+So, let's say that in our case, we are only interested in the first 3 rows of the spreadsheet, or we are interested if their last name is `'Wick'` (you've seen what he can do when he is upset).
+
+For this, our logic would look something like this:
+
+1. `'Has Access?'` = `'Yes'` **and** `Row Number` < 4
```

Comment:
btw is this possible? 

`'Has Access?'` = `'Yes'` **and** (`Row Number` < 4 **or** `'Last Name'` = `'Wick'`)

if not, make sure to mention that you cannot combine 3 constraints at the same time, but you'd have to do them in pairs (although this is not very scalable)

Reply:
you can only link `AND` conditions like this:

c1 AND c2 AND c3 AND ... c100
OR
c1 AND d1 AND d2 AND ... d100